### PR TITLE
Zbh edits

### PR DESCRIPTION
**Pull request by aguynamedryan**

### CODE REVIEW

File: dat/missions/zalek/blackhole/zbh01.lua [hunk 1]
```diff
@@ -108,26 +108,26 @@ function land ()
...
+   vn.na(fmt.f(_("You eventually make your way around the entire station and end up back at the docks, near your ship, {shipname}. Zach seems to be staring at some of the of the damage. As you struggle with how to break the silence, you notice that Zach has become fixed on some structural damage."), {shipname=player.pilot():name()} ))
    z(_([["Say, doesn't that over there look like bite marks?"]]))
    vn.na(_([[You look over at where Zach is pointing, but can't really make out what he's getting at. However, after a while of staring and squinting, you seem to make out some damage that could indeed be bite marks, but what on earth would bite a station?]]))
    z(_([["Something isn't right here."
 He stops one second realizing what he just said and gives a puzzled look.
 "Well, besides the obvious, it just doesn't make sense. Why would anyone want to do this to a basic research laboratory? There is nothing over here other than the black hole…"]]))
-   z(_([[He goes silent and seems to be lost in thought. Finally, he looks up and looks around the ship, his gaze becoming determined like rocks after the snow melts away. He then turns to you.
```
Comment (bobbens) [inline dat/missions/zalek/blackhole/zbh01.lua:123]: RIP my attempt at being poetic.

Comment (not-pyroman) [reply dat/missions/zalek/blackhole/zbh01.lua:123]: Sad

**Comment by UncombedCoconut** [comment]
Hmm, well, let's merge the first commit, then figure out if "RIP my attempt at being poetic" is something we want to happen?

**Comment by bobbens** [comment]
I would be fine with merging it all. As it's still a WIP campaign, I'll probably do a second writing pass when it's done to change the tone of some things and try to enforce more internal consistency.

**Pull request merged**